api/src/main/resources/workbench-api.yaml

```diff
@@ -1079,6 +1079,31 @@ paths:
           description: User doesn't have the ACCESS_CONTROL_ADMIN authority
           schema:
             "$ref": "#/definitions/ErrorResponse"
+  "/v1/admin/users/{userId}/updateVerifiedInstitutionalAffiliation":
+    parameters:
+    - "$ref": "#/parameters/userId"
+    post:
+      tags:
+      - profile
+      consumes:
+      - application/json
+      description: Updates a user's verified institution. Requires ACCESS_CONTROL_ADMIN authority.
```

Comment:
Nit: it might be worth mentioning here that institutional validation is executed by this API method, e.g. a request will fail if the indicated user is not a valid member of the new affiliated institution.